app/code/core/Mage/Core/Helper/EnvironmentConfigLoader.php

```diff
@@ -14,17 +14,23 @@
  */
 class Mage_Core_Helper_EnvironmentConfigLoader extends Mage_Core_Helper_Abstract
 {
-    protected const ENV_STARTS_WITH = 'OPENMAGE_CONFIG';
-    protected const ENV_FEATURE_ENABLED = 'OPENMAGE_CONFIG_OVERRIDE_ALLOWED';
-    protected const ENV_KEY_SEPARATOR = '__';
-    protected const CONFIG_KEY_DEFAULT = 'DEFAULT';
-    protected const CONFIG_KEY_WEBSITES = 'WEBSITES';
-    protected const CONFIG_KEY_STORES = 'STORES';
+    public const ENV_STARTS_WITH = 'OPENMAGE_CONFIG';
```

Comment:
Why are these public now? Does it help when it gets a class rewrite?

Reply:
imho the constants should be availble from outside.

Reply:
No special purpose atm. All constants in helpers are public, why not here?

Reply:
Hm I am not seeing it anymore. Its okay to change visibility, if you think its correct. Or maybe we should move away from constants all together, since it makes it too complicated. No other tooling should need to use these constants, so they could be public methods just the same.

Or?